deps.qt/qt6.ps1

```diff
@@ -12,6 +12,7 @@ $QtComponents = @(
     'qtshadertools'
     'qtmultimedia'
     'qtsvg'
+    'qtcharts'
```

Comment:
Does qtcharts require anything other than qtbase? If not, then it should be placed just below qtbase.

Reply:
you're right : https://github.com/qt/qtcharts/blob/dev/dependencies.yaml
But for CI on linux, I noticed that CI failed if I didn't add qtopenGL and qtopenGLWidgets in CI/linux/01_install_dependencies.sh

Reply:
That's strange. Though, that seems like a Linux-only concern, unless we can confirm that it actually builds these components on Windows or macOS.

Reply:
positioned qtcharts just below qtbase

Reply:
> That's strange. Though, that seems like a Linux-only concern, unless we can confirm that it actually builds these components on Windows or macOS, so I'm not sure "and its dependencies" needs mentioned.

good point. I removed qtopengl and qtopenglwidgets but had forgotten to update the commit message to my utter shame 🤦